web/app/help/operations/export-edge-attributes-to-neo4j.asciidoc

```diff
@@ -0,0 +1,38 @@
+### Export edge attributes to Neo4j
+
+Exports edge attributes from a graph in LynxKite to a
+corresponding graph in https://neo4j.com/[Neo4j].
+
+The relationships in Neo4j are identified by a key property (or properties).
+You must have a corresponding edge attribute in LynxKite by the same name.
+This will be used to find the right relationship to update in Neo4j.
+
```

Comment:
I think we need to explicitly spell out what happens if multiple edges match a given key value combination
a) in Neo4j
b) in LynxKite

Reply:
> I think we need to explicitly spell out what happens if multiple edges match a given key value combination
> a) in Neo4j
> b) in LynxKite

Done. I've also added the full Cypher query to the docs. I imagine this may be of interest to advanced Neo4j users and it's not too scary.